src/tools/dev/masonry/opts/mb-trunk-darwin-10.11-x86_64.json

```diff
@@ -26,4 +25,4 @@
                  "pyside",
                  "xdmf",
                  "zlib"]}
-}}
\ No newline at end of file
+}}
```

Comment:
I am inclined to leave these older config files unchnaged for posterity's sake. So, maybe not change this one or the `10.6` one or `linux-x86_64`.

Reply:
which linux-x86_64 file?